### ISSUE TITLE
Colour contrast in code blocks is not ideal

### ISSUE THREAD

**Issue by hturner** [rjournal/rjtools#95]
Picked up with Accessible Web Helper extension:

Colour contrast in code blocks fails check "Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds."

The colours already seem quite dark, maybe a slightly bigger font would help so the coloured parts are slightly thicker?

**Comment by mitchelloharawild** [comment]
Could you point me to the code block which has the contrast issue? We can consider switching to a different code highlighting style.
